samples/deployment/vmware/README.md

```diff
@@ -0,0 +1,69 @@
+# Setup Guide
+
+## Setup dependencies
+
+1. Create virtual env
+    ```bash
+    virtualenv -p python3.7 ./venv
```

Comment:
Virtualenv should be previously installed.

Reply:
Let's use `venv` as it comes as a part of the standard library since Python 3.3. I guess, its functionality should be enough for us. Essentially, it's the same, but `venv` becomes a de-facto standard in Python3.
```
python3 -m venv ./venv
```
https://packaging.python.org/tutorials/installing-packages/#creating-virtual-environments